templates/v3/includes/_empty_state.html

```
@@ -0,0 +1,33 @@
+                                            "page" (full-page, larger headline)
+    cta_label          (string, optional) - call-to-action label
+    cta_url            (string, optional) - call-to-action destination, built in the view
+
+  Usage:
+    {% include "v3/includes/_empty_state.html" with title="No results, please search again..." description="Try a shorter keyword, or check the spelling." variant="inset" only %}
+{% endcomment %}
+<div class="empty-state{% if variant %} empty-state--{{ variant }}{% endif %}">
+  {% if title_is_heading %}
+    <h1 class="empty-state__title">{{ title }}</h1>
+  {% else %}
+    <p class="empty-state__title"{% if title_x_text %} x-text="{{ title_x_text }}"{% endif %}>{{ title }}</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)